aries-core/src/js/components/core/Tiles/Tiles.js

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+
+import { Grid } from 'grommet';
+
+export const Tiles = ({ ...rest }) => (
+  <Grid columns={{ count: 'fit', size: 'small' }} {...rest} />
```

Comment:
Just out of curiosity, what is the reason that we chose this as the default size for the columns?

Reply:
Chris's designs.